packages/react-renderer-demo/src/components/landing-page/landing-page-cards.js

```diff
@@ -295,6 +295,16 @@ const LandingPageCards = () => {
                 </a>
               </RouterLink>
             </Grid>
+            <Grid item xs={12} md={4} xl={2}>
```

Comment:
The previous two icons should be updated and have `md` set to `4` to make a row.